.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature Request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+------------------------------- delete below -------------------------------
+
+Thank you for submitting a feature request! A few things:
+
+- Please check that a similar feature request has not already been submitted. Adding support to a currently existing feature request by commenting will increase the visibility and importance of your request beyond submitting a duplicate request. 
+
+- If this is a large feature request, please submit it to user voice.
+https://feedback.azure.com/forums/321918-azure-iot
+
+As an open source project, we always welcome PRs for new features. So if you know what needs to be done, feel free to create the change and send it our way.
```

Comment:
We should provide best practices for submitting PR's.  In addition to Yi's suggestion, the code will require unit testing, potentially e2e tests, and potentially documentation changes.  If the customer has any questions on setting this up or what we think set of tests should be, they should ask when opening the issue with proposed solution.

Reply:
Also they should follow coding conventions in the file(s) they're modifying when doing a pr.

Reply:
I've edited this line to reflect asking them first to submit an issue so we can guide them on best practices for a PR.